src/test/java/ie/gmit/ItemTest.java

```diff
@@ -0,0 +1,40 @@
+package ie.gmit;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+public class ItemTest {
+    private Item i1;
+
+    @BeforeEach
+    void init(){i1 = new Item("Name", 3.33, 14);}
+
+    @Test
+    public void testItemCreateSuccess(){
+        Item item = i1;
```

Comment:
Dont think this line is needed and the whole test can just be the assertEquals because you're making an item in init() and making another here